src/cmd/services/m3query/scripts/curlDiff.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script can be used to compare prometheus query output with m3query output
```

Comment:
Perhaps making this into a function so that it could be sourced into other scripts in the future. Just a thought

Reply:
that's a good idea but I'm not sure how useful it would be to call it as a function vs calling this script completely and then working off generated files. Also, note that this is a simple script which just generates output from prom and m3. We'll be working on a more complex way to compare prom output with m3 output.